src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -145,10 +145,27 @@ public void addAuthor(Author author, List<String> ignoreGlobList) {
     public void removeIgnoredAuthors(List<String> ignoredAuthorsList) {
         for (String author : ignoredAuthorsList) {
             if (authorEmailsAndAliasesMap.containsKey(author)) {
-                authorList.remove(authorEmailsAndAliasesMap.get(author));
+                removeAuthorInformation(author);
             }
         }
-        resetAuthorInformation();
```

Comment:
I noticed that `resetAuthorInformation()` is used in `RepoConfiguration#setAuthorList`, could we utilise your `removeAuthorInformation` instead?

Reply:
I think `removeAuthorInformation` is not a direct substitute for `resetAuthorInformation`, since resetting also involves re-building the internal maps in author config for the authors in the author list.

Reply:
ok noted

Reply:
As we discussed in issue #1204, the current naming of `resetAuthorInformation` is a bit confusing. We may break it into 2 methods: `clear` and `buildFromAuthorList` (as mentioned in the isue)